Spatial.gd

```diff
@@ -372,9 +365,28 @@ func reset_minimap_masks():
 		compass_corner1 = viewport_size - Vector2(compass_width, compass_height)
 		compass_corner2 = compass_corner1 + Vector2(compass_width, compass_height)
 
-	for minimap_path in $Control/MiniMap.get_children():
-		minimap_path.material.set_shader_param("minimap_corner", compass_corner1)
-		minimap_path.material.set_shader_param("minimap_corner2", compass_corner2)
+	for child in $Control/MiniMap.get_children():
+		if child.get_name() == "Category2D":
```

Comment:
`Control/MiniMap` on startup has a child node called `Line2D`. It's possible that it can be deleted since we are rebuilding  Minimap but I did not explore the ramifications of removing it.